Stellar-operation-manage-asset.x

```diff
@@ -98,37 +123,62 @@ struct ManageAssetOp
 
 /******* ManageAsset Result ********/
 
+//: Result codes of ManageAssetOp
 enum ManageAssetResultCode
 {
-    // codes considered as "success" for the operation
+    //: Specified action in `data` of ManageSignerOp was successfully performed
     SUCCESS = 0,                       // request was successfully created/updated/canceled
 
     // codes considered as "failure" for the operation
-	REQUEST_NOT_FOUND = -1,           // failed to find asset request with such id
-    INVALID_SIGNATURE = -2,           // only asset pre issuer can change asset pre issuer
-	ASSET_ALREADY_EXISTS = -3,	      // asset with such code already exist
+    //: There is no `CREATE_ASSET` or `UPDATE_ASSET` request with such id
+    REQUEST_NOT_FOUND = -1,           // failed to find asset request with such id
+    //: only asset pre issuer can manage asset
+    INVALID_SIGNATURE = -2,
+    //: Not allowed to create asset with code which already used for an another asset
+    ASSET_ALREADY_EXISTS = -3,	      // asset with such code already exist
+    //: Not allowed to set max issuance amount that
```

Comment:
//: It is not allowed to set max issuance amount that is
    //: less than the sum of issued, pending issuance and available for issuance values
?